web-ui/src/streaming/inspection/QuantilePagination.tsx

```diff
@@ -0,0 +1,48 @@
+// Adds a slider for quantile selection to the regular data-grid pagination
```

Comment:
Agreed, I also did some experiments with infinite scrolling approaches instead of the pagination which would be ideal... Unfortunately, I think for a non-confusing UX with this kind of approach you'd need what we discussed a month or two ago (e.g., store view in a DB on backend) so we can filter/sort the tables easily.